oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -181,6 +181,8 @@ public class DocumentNodeStoreService {
      */
     private static final String FT_NAME_DOC_STORE_THROTTLING = "FT_THROTTLING_OAK-9909";
 
+    private static final String FT_NAME_DOC_STORE_NOCOCLEANUP = "FT_NOCOCLEANUP_OAK-10660";
```

Comment:
```suggestion
    private static final String FT_NAME_DOC_STORE_COCLEANUP = "FT_COCLEANUP_OAK-10660";
```

Reply:
I'll add that feature toggles should be short-lived (12 months?). As such it's not that important how we call them.